src/Mutation/CommentCreate.php

```diff
@@ -27,7 +27,7 @@ public static function register_mutation() {
      */
     public static function get_input_fields() {
         return [
-            'postId'      => [
+            'commentOn'      => [
                 'type'        => 'Int',
                 'description' => __( 'The ID of the post the comment belongs to.', 'wp-graphql' ),
```

Comment:
slight change, but we should update this description to say the ID of the `post object` the comment belongs to. 

Since it isn't always necessarily a "post"